tutorials/ner_double/project.yml

```diff
@@ -0,0 +1,50 @@
+title: "Combining Multiple Trained NER Components"
+description: > 
+  This project shows you the different ways you can combine multiple trained
+  NER components and their tradeoffs.
+
+  Note that before running this project, you should run the
+  [ner_drugs](../ner_drugs) project and install the packaged version of it.
+
+vars:
+  name: "ner_double"
+  version: "0.0.0"
+  drug_model: "ner_drugs-0.0.0"
+
+
+# TODO is there a better name than "training" for assembled packages?
+directories: ["configs", "scripts", "training", "packages"]
+
+workflows:
+  all:
+    - assemble
```

Comment:
To be complete, perhaps this `project.yml` should also define the `download` command for the pretrained spacy pipeline, and define `en_ner_drugs` as an asset? Or where are you supposed to get the latter from? (maybe I missed it)

Reply:
My expectation was that a user would check out this repo, build the drug model separately and install it, and then use this tutorial. That's stated at the top of the README, but is there some way I can automate it in the `project.yml`? I guess I could make it cd to the other dir and call `spacy project` there but that feels weird...

Reply:
No I wouldn't do that, but I was thinking maybe we could provide a prebuilt drug model somewhere that is just downloadable as an asset?

Reply:
Ah, makes sense.

Reply:
I made a dummy release and added the drug model as an attachment, so it's an asset now. Much cleaner!